tests/Storefront/Controller/SearchControllerTest.php

```diff
@@ -81,4 +81,76 @@ public function testAvailableFilterReturnsCorrectResponse(string $demoResponse,
 
         $this->assertSame($filterResponse, $expectedFilters);
     }
+
+    public function testFiltersWhichAreNotInTheAvailleFilterResponseAreStillReturned(): void
+    {
+        if (Utils::versionLowerThan('6.3.3.0')) {
+            $this->markTestSkipped('Filter disabling feature was introduced in Shopware 6.3.3.0');
+        }
+
+        $allFiltersResponse = new Xml21Response($this->getMockResponse('XMLResponse/demoResponseWithNoResults.xml'));
+        $responseParser = new Xml21ResponseParser($allFiltersResponse);
+        $availableFilters = $responseParser->getFiltersExtension();
+
+        $response = new Xml21Response($this->getMockResponse('XMLResponse/demoResponseWithAvailableFilters.xml'));
```

Comment:
`demoResponseWithAvailableFilters.xml` is used for `$allFilters`? Please try to improve naming.

Reply:
This response is used by other tests as well. Now using `demoResponseWithAllFilterTypes.xml` instead, to make this clearer.